### DISCUSSION TITLE
Fix website

### DISCUSSION THREAD

**Discussion by HGT-123** [AspectLinux/aspectlinux.github.io#7] [General]
Can you please add more contributors to the about page of the website. My own username (HGT-123) is not there :(

**Comment by udscminidisc** [comment]
Got it.


Sincerely, yooit_32<https://scratch.mit.edu/users/yooit_32/>.
​
________________________________
From: HGT-123 ***@***.***>
Sent: Tuesday, January 10, 2023 9:48 PM
To: AspectLinux/aspectlinux.github.io ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [AspectLinux] Fix website (Discussion #7)


Can you please add more contributors to the about page of the website. My own username (HGT-123) is not there :(

**Reply by HGT-123** [reply]
Thx
